src/lib/config/common.js

```diff
@@ -0,0 +1,29 @@
+// Temporary extracted common props that used in init
+
+export const DEFAULT_CURRENCY = 'CHF';
```

Comment:
I don't know if it was discussed, but it is not supposed to be config part anymore? Maybe I'm missing something

Reply:
resolved. it is implemented this way to avoid duplicating in case of changes